src/nanorc/cfgmgr.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, log, config_url, process_manager_description, port_offset=0,
         self.boot = {}
         self.port_offset = port_offset
         self.scheme = None
-        self.expected_std_cmds = ['init', 'conf']
+        self.expected_std_cmds = ['init', 'conf', 'boot', 'daqconf_multiru_gen', 'dromap', 'config']
```

Comment:
Can that be renamed to `ignore_for_custom_cmd` or so?

Reply:
Done